src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/FixAbstractMethodsStep.cs

```diff
@@ -281,11 +281,6 @@ void AddNewExceptionMethod (TypeDefinition type, MethodDefinition method)
 			}
 		}
 
-		public virtual void LogMessage (string message, params object [] values)
```

Comment:
This method is overridden in https://github.com/xamarin/xamarin-android/blob/master/src/Xamarin.Android.Build.Tasks/Tasks/LinkAssembliesNoShrink.cs#L108